scripts/upload.gradle

```diff
@@ -1,198 +1,7 @@
-/*
```

Comment:
hmm, this split may conflict with #2426, I'll resolve the changes requested there this week!

Reply:
Resolved I think.

Reply:
Looks good.